### DISCUSSION TITLE
Missing punctuation marks `?!` ?

### DISCUSSION THREAD

**Discussion by pataxis** [sapegin/textlint-rule-terminology#76] [Bugs] [answered]
**textlint**: v13.4.1

Not sure if it's a bug or if there's a reason, but I noticed that [Single words](https://github.com/sapegin/textlint-rule-terminology/blob/master/terms.jsonc#L105) are not detected with trailing punctuation marks `?` and `!`. Could be the same for all "replacements".

```md
This is found: css?
This isn't: file-name?
Neither are these: user-base! server side!
```
This seems similar to #28 and fix [18cfb94](https://github.com/sapegin/textlint-rule-terminology/commit/18cfb94c3eb4de7506a318ee6cbc701236f83388), would it be worth to add `?!` to the list of punctuation characters?

**Comment by sapegin** [comment]
Yeah, this is definitely a bug.

**Comment by sapegin** [comment, marked as answer]
Done in https://github.com/sapegin/textlint-rule-terminology/commit/5b5165a03fb85aa1367e89889e5c47b7d8d131fc
